### DISCUSSION TITLE
Can't display Github Actions UI logs after refreshing the page when using self-hosted runner

### DISCUSSION THREAD

**Discussion by iliasnaamane2** [community/community#118795] [Actions]
### Select Topic Area

Bug around Github Actions logs

### Body

 The issue doesn't occur with Github runners, it occur only with self-hosted-runners. After running a workflow, I can see the logs but strangely disappeared after refreshing the page. Please see screenshot below showing that I can't access logs:

![image](https://github.com/community/community/assets/155658414/e0e1f3ce-0e9e-41da-99ef-550708026515)

Thanks in advance for your help!

**Comment by marchaarsma** [comment]
I am seeing the same thing happen on a self-hosted linux action-runner. A similar task on a windows system is working fine. When looking at view-source i notice that the data-log-url="" of all check-step are empty. On the windows system they contain a valid URL . The problem is also described here originaly [https://github.com/actions/actions-runner-controller/issues/3406](https://github.com/actions/actions-runner-controller/issues/3406) , and for us too after downloading log archive the zip file is empty.

**Reply by 631117HK** [reply]
That's right
